src/klein/test/test_plating.py

```diff
@@ -1,19 +1,29 @@
 """
 Tests for L{klein.plating}.
 """
-
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
```

Comment:
This duplicates the next statement.

Reply:
Whoops - made a mistake reverting this :( I chose the one that minimizes the diff against master.

Thanks!